src/MyShop/WebsiteBundle/Resources/views/Index/common.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::base.html.twig' %}
+{% import 'WebsiteBundle:Index:macro.html.twig' as macro %}
+
+{% block body %}
+    {% block menu %}
+        <div class="menu">
+            {{ macro.show_menu() }}
```

Comment:
Please render menu, and all common blocks in `base.html.twig`. Keep specific blocks only for templates in bundles